projects/js-packages/publicize-components/src/components/share-status/shares-dataview.tsx

```diff
@@ -0,0 +1,99 @@
+import { DataViews } from '@wordpress/dataviews';
+import { getDate, humanTimeDiff } from '@wordpress/date';
+import { __ } from '@wordpress/i18n';
+import { PostShareStatus, ShareStatusItem } from '../../social-store/types';
+import ConnectionIcon from '../connection-icon';
+import { ShareStatusAction } from './share-status-action';
+import { ShareStatusLabel } from './share-status-label';
+import styles from './styles.module.scss';
+
+const getItemId = ( item: ShareStatusItem ) => {
+	return `${ item.external_id || item.connection_id }:${ item.timestamp }`;
+};
+
+const noop = () => {};
+
+type SharesDataViewProps = {
+	postShareStatus: PostShareStatus;
+};
+
+/**
+ * The component for the shares data view.
+ *
+ * @param {SharesDataViewProps} props - The component props.
+ *
+ * @return {import('react').ReactNode} - The shares data view component.
+ */
+export function SharesDataView( { postShareStatus }: SharesDataViewProps ) {
+	return (
+		<div className={ styles[ 'dataview-wrapper' ] }>
+			<DataViews
+				isLoading={ postShareStatus.loading }
+				getItemId={ getItemId }
+				fields={ [
+					{
+						id: 'connection',
+						label: __( 'Connection', 'jetpack' ),
+						render: ( { item } ) => (
+							<div className={ styles[ 'connection-name' ] }>
+								<ConnectionIcon
+									serviceName={ item.service }
+									label={ item.external_name }
+									profilePicture={ item.profile_picture }
+								/>
+								<div className={ styles[ 'share-item-name-wrapper' ] }>
+									<div className={ styles[ 'share-item-name' ] }>{ item.external_name }</div>
+								</div>
+							</div>
+						),
+						enableSorting: false,
+						enableHiding: false,
+					},
+					{
+						id: 'timestamp',
+						label: __( 'Time', 'jetpack' ),
```

Comment:
I think it would be better to call this `Shared at` or something similar

Reply:
To me, time feels too vague, also I looked up on the designs it's 'Published at' on the admin page, maybe that's where I got the idea from :D I think Shared at still makes more sense than published at. Either way we should use the same term here and there in the future

Reply:
But what about the case of failure? We can't say "Shared at".

Reply:
That's a point we have to make on the admin page designs too, but let's leave time now then